mem-01-b.xml

```diff
@@ -56,7 +56,7 @@
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{naH:n}</column>
-      <column name="notes">This refers to a kind of plant, rather than the finely-ground spice produced from such plants.[1]</column>
```

Comment:
Removed because black pepper comes from a different plant (Piper nigrum) which isn't closely related to bell peppers (Capsicum annuum).